server/lib/db/index.js

```diff
@@ -28,6 +28,7 @@ class DatabaseConnector {
   /** @constructor */
   constructor() {
     const params = {
+      multipleStatements: true,
```

Comment:
I wondered how you had executed multiple statements above.  Please don't add this option - it slows down `node-mysql`'s query processing significantly.  It also opens us up to more types of SQL injection vulnerabilities, as warned in [the documentation](https://github.com/mysqljs/mysql#connection-options).